lib/api/remoteActions/cleanAndPrepare.js

```diff
@@ -39,7 +46,6 @@ timeOutCB = () => {
   console.log(error('Can\'t contact Kuzzle'));
   process.exit(1);
 };
-
```

Comment:
nitpicking: shouldn't it be cleaner to keep an empty line before `module.exports...` affectation ?